deepforest/scanner.py

```diff
@@ -0,0 +1,58 @@
+"""
+Implement methods on extracting structured data such as images or sequences.
+"""
+import torch
+import torch.nn as nn
```

Comment:
This will make `torch` a hard dependency for Deep Forest. We'd better use the `__import__` statement. You can refer to [here](https://github.com/LAMDA-NJU/Deep-Forest/blob/master/deepforest/cascade.py#55) for an example.